app/src/main/java/fr/free/nrw/commons/upload/DescriptionsAdapter.java

```diff
@@ -229,7 +229,17 @@ private void initLanguageSpinner(int position, Description description) {
                     int defaultLocaleIndex = languagesAdapter.getIndexOfUserDefaultLocale(context);
                     spinnerDescriptionLanguages.setSelection(defaultLocaleIndex);
                 } else {
-                    spinnerDescriptionLanguages.setSelection(0);
+                    int availableLangIndex = -1;
```

Comment:
Can you please add comments to your code, so that we can understand what is happening

Reply:
@neslihanturan  Sure :)